src/Components/Patient/PatientRegister.tsx

```diff
@@ -1303,6 +1308,14 @@ export const PatientRegister = (props: PatientRegisterProps) => {
                               }}
                               types={["mobile", "landline"]}
                             />
+                            <Checkbox
+                              label="Is the phone number an emergency number?"
+                              className="font-bold"
+                              checked={isEmergencyNumberEnabled}
+                              onCheck={(checked) =>
+                                setIsEmergencyNumberEnabled(checked)
+                              }
```

Comment:
Instead update the form State (update the emergency phone number in the form state with the phone number here).